### DISCUSSION TITLE
Question about contribution

### DISCUSSION THREAD

**Discussion by Tusenka** [pytest-dev/pytest#13000] [Q&A] [answered]
Intro:
I begin to learn project
Question:
Why here we take only fixturedefs args for the last parent?
https://github.com/pytest-dev/pytest/blob/acf130311806cb1a627ea0e21369c7d6f586663a/src/_pytest/fixtures.py#L1599

**Comment by RonnyPfannschmidt** [comment, marked as answer]
I believe that place is for finding the closest fixture definition 

That's the last in the list 

This plays a role for fixtures that expand an fixture of the same name
